news/2024-07-01-rocky-linux-9-cve-2024-6378-regression.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Rocky Linux 9 - CVE-2024-6387: regreSSHion
```

Comment:
Looks like it's failing due to a colon in the frontmatter title. It's YAML, so you'll need to escape it.